plextraktsync/plex_api.py

```diff
@@ -607,16 +608,17 @@ def rate(self, m, rating):
         m.rate(rating)
 
     @nocache
-    def create_playlist(self, name: str, items):
-        _, plex_items_sorted = zip(*sorted(dict(reversed(items)).items()))
-        self.plex.createPlaylist(name, items=plex_items_sorted)
-
-    @nocache
-    def delete_playlist(self, name: str):
+    def update_playlist(self, name: str, items: List[Union[Movie, Show, Episode]]) -> None:
+        """
+        Updates playlist (creates if name missing) replacing contents with items[]
+        """
         try:
-            self.plex.playlist(name).delete()
-        except (NotFound, BadRequest):
-            logger.debug(f"Playlist '{name}' not found, so it could not be deleted")
+            playlist: Playlist = self.plex.playlist(name)
+        except NotFound:
+            playlist = self.plex.createPlaylist(name)
```

Comment:
The createPlaylist method requires items to be passed.
Right now it's giving a BadRequest error when creating a new playlist.
I can check to make a PR for this as well.

Reply:
oh ok, sorry my bad for not testing that use case, does it accept an empty list `[]`?

luckily this code is not yet released (no tag yet), so no harm done.

Reply:
> oh ok, sorry my bad for not testing that use case, does it accept an empty list `[]`?

It does not apparently. That sucks, because if a user likes a list for which he has no mutual items, the script will crash.
We'll need to add a check if the list has items. 
I'll make a new PR for this.